src/dataflow-types/src/client/controller.rs

```diff
@@ -122,20 +123,32 @@ where
                 let service = orchestrator
                     .namespace("compute")
                     .ensure_service(
-                        &format!("instance-{instance}"),
+                        &format!("cluster-{instance}"),
                         ServiceConfig {
                             image: dataflowd_image.clone(),
                             args: vec![
                                 "--runtime=compute".into(),
                                 format!("--storage-addr={storage_addr}"),
-                                "0.0.0.0:2101".into(),
+                                "0.0.0.0:2102".into(),
+                            ],
+                            ports: vec![
+                                ServicePort {
+                                    name: "controller".into(),
+                                    port: 2100,
+                                },
+                                ServicePort {
+                                    name: "compute".into(),
+                                    port: 2102,
```

Comment:
These ports are duplicated all over. Maybe it would be good to make them into constants, so the type system can reduce the risk of typos.

Reply:
Ack. I filed https://github.com/MaterializeInc/materialize/issues/11615 to track, but leaving as is for now.